### DISCUSSION TITLE
move-up property is not working

### DISCUSSION THREAD

**Discussion by vikas-growisto** [fastn-stack/fastn#581] [Q&A]
-- ftd.text: FifthTry
move-up: 20
;;move-down: 20
;;move-left: 20
;;move-right: 20

I tried the properties move-up but it is not working.
Similarly move-down, move-left and move-right

**Comment by Arpita-Jaiswal** [comment]
These properties are yet not supported.
Please subscribe to [this issue](https://github.com/ftd-lang/ftd/issues/591) to get updates and notifications.
